doc/Core.xml

```diff
@@ -5352,6 +5370,592 @@ onvif://www.onvif.org/name/ARV-453
+      <title>Security Parameter Configuration</title>
+      <para>
+        This section details the requirements for authentication and privacy protocols supported by the SNMPv3 extension.
+        Devices <emphasis role="bold">shall</emphasis> securely store localized keys and apply them according to the user's configured security level.
+        Keys for authentication and privacy <emphasis role="bold">shall</emphasis> be provided by the Client as base64-encoded binary data using the <literal>tt:BinaryData</literal> type. The Device <emphasis role="bold">shall</emphasis> delocalize these keys using its EngineID as per SNMPv3 USM specifications (<xref linkend="RFC3414"/> Appendix A) before storing them internally.
+      </para>
+      <section xml:id="s.snmpv3_auth_protocols">
+        <title>Authentication Protocols</title>
+        <para>
+          The following authentication protocols <emphasis role="bold">shall</emphasis> be supported:
+        </para>
+        <itemizedlist>
+          <listitem>
+            <para>
+              <emphasis role="bold">HMAC-SHA-256:</emphasis> As specified in <xref linkend="RFC7860"/> (OID: <literal>usmHMAC192SHA256AuthProtocol</literal>). This protocol is <emphasis role="bold">REQUIRED</emphasis>. The authentication key length is 32 octets.
+            </para>
+          </listitem>
+        </itemizedlist>
+        <para>
+          Support for stronger authentication algorithms is <emphasis role="bold">RECOMMENDED</emphasis>:
+        </para>
+        <itemizedlist>
+          <listitem>
+            <para>
+              <emphasis role="bold">HMAC-SHA-384:</emphasis> As specified in <xref linkend="RFC7860"/> (OID: <literal>usmHMAC256SHA384AuthProtocol</literal>). The authentication key length is 48 octets.
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <emphasis role="bold">HMAC-SHA-512:</emphasis> As specified in <xref linkend="RFC7860"/> (OID: <literal>usmHMAC384SHA512AuthProtocol</literal>). The authentication key length is 64 octets.
+            </para>
+          </listitem>
+        </itemizedlist>
+        <para>
+          The corresponding <literal>tt:SNMPAuthenticationProtocol</literal> enumeration values would be <literal>HMAC_SHA_256</literal>, <literal>HMAC_SHA_384</literal>, and <literal>HMAC_SHA_512</literal>.
+        </para>
+      </section>
+      <section xml:id="s.snmpv3_priv_protocols">
+        <title>Privacy Protocols</title>
+        <para>
+          The following privacy protocols <emphasis role="bold">shall</emphasis> be supported:
+        </para>
+        <itemizedlist>
+          <listitem>
+            <para>
+              <emphasis role="bold">AES-128-CFB:</emphasis> As specified in <xref linkend="RFC3826"/> (OID: <literal>usmAesCfb128Protocol</literal>). This protocol is <emphasis role="bold">REQUIRED</emphasis>. The privacy key length is 16 octets.
```

Comment:
@HansBusch do we need to also support these protocols "AES-128-CFB" in the Security Baseline doc? #571